### DISCUSSION TITLE
Add option to make organization teams visible to general public

### DISCUSSION THREAD

**Discussion by hermidalc** [community/community#20808] [Other Feature Feedback, Questions, & Ideas]
Would be nice to have an option to make organization teams visible to the general public, not just org visible or secret. Then on the organization overview home page the Teams tab would be visible and general public could drill down to specific team overview pages, their details, and members.

**Comment by jfullstackdev** [comment]
you mean, the member does not have the option to hide his / her membership ? I think it would violate privacy

**Comment by hermidalc** [comment]
I imagine you can still let members hide their membership if they want to, but still allow the option for admins etc to make the team visible to the general public, and those members inside the team that do not want to hide their membership are then visible.

In my academic organization, the UPMC Tumor Microenvironment Center, we have multiple laboratories that naturally map to GitHub teams and that's what I've done. But we want the option to make these teams visible to the public.
